Cargo.toml

```diff
@@ -8,8 +8,12 @@ name = "noodles"
 doctest = false
 
 [dependencies]
-docopt = "0.6.66"
-docopt_macros = "0.6.66"
+docopt = "0.6.69"
+# docopt_macros = "0.6.69" When docopt_macros cuts a new release this can come back
 rustc-serialize = "0.3.15"
 curl = "0.2.10"
-yaml-rust = "0.2.0"
+yaml-rust = "0.2.1"
+
+[dependencies.docopt_macros]
+git = "https://github.com/docopt/docopt.rs.git"
+rev = "ba5cf65299c5bf4a67ae7c8efae2791a19f612c4"
```

Comment:
This didn't compile for me... 

```
root@dev-big:~/progs/noodles# cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/docopt/docopt.rs.git`
 Downloading docopt v0.6.69
   Compiling docopt v0.6.69 (https://github.com/docopt/docopt.rs.git?rev=ba5cf65299c5bf4a67ae7c8efae2791a19f612c4#ba5cf652)
   Compiling docopt v0.6.69
   Compiling docopt_macros v0.6.69 (https://github.com/docopt/docopt.rs.git?rev=ba5cf65299c5bf4a67ae7c8efae2791a19f612c4#ba5cf652)
/root/.cargo/git/checkouts/docopt.rs-81b31a115c87e3e0/ba5cf65299c5bf4a67ae7c8efae2791a19f612c4/docopt_macros/src/macro.rs:262:29: 262:37 error: mismatched types:
 expected `syntax::ast::Name`,
    found `&'static str`
(expected struct `syntax::ast::Name`,
    found &-ptr) [E0308]
/root/.cargo/git/checkouts/docopt.rs-81b31a115c87e3e0/ba5cf65299c5bf4a67ae7c8efae2791a19f612c4/docopt_macros/src/macro.rs:262         if deriving.name != "derive" {
                                                                                                                                                          ^~~~~~~~
/root/.cargo/git/checkouts/docopt.rs-81b31a115c87e3e0/ba5cf65299c5bf4a67ae7c8efae2791a19f612c4/docopt_macros/src/macro.rs:262:29: 262:37 help: run `rustc --explain E0308` to see a detailed explanation
error: aborting due to previous error
Could not compile `docopt_macros`.

To learn more, run the command again with --verbose.
```

Using `docopt_macros = "0.6.69"` worked just fine though.